.github/workflows/release.yml

```diff
@@ -32,3 +32,18 @@ jobs:
           workflow: publish.yml
           ref: master
           inputs: '{ "acvm-ref": "${{ needs.release-please.outputs.tag-name }}" }'
+
+  dispatch-acvm-js-wasm:
+    name: Dispatch to acvm-js-wasm
+    needs: [release-please]
+    if: ${{ needs.release-please.outputs.tag-name }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Dispatch to acvm-js-wasm
+        uses: benc-uk/workflow-dispatch@v1
+        with:
+          workflow: update.yml
+          repo: noir-lang/acvm-js-wasm
```

Comment:
This repo doesn't exist. We should rename acvm_simulator_wasm.

Reply:
Yes, let's rename

Reply:
done